### ISSUE TITLE
Problem when reproducing the result

### ISSUE THREAD

**Issue by PinTang1999** [aim-uofa/DyCo3D#16]
Thank you for your code. When I trained the model using Scannet v2 on my server (batch size 16, four 3090 GPUs), I found a reduction on the peformance, which is shown below. When I tried to load your trained model for evaluation, I realized that your model is trained using 50k iterations. So, I changed the iter to 5k in the config file. However, the results is still unsatisfactory. Could you give more details about your experiments to produce the results, like the number of prepare iterations using PointGroup (i.e., the training itr for semantic segmentation), batch size and corresponding learning rate? Thank you!
![image](https://user-images.githubusercontent.com/66257799/158041653-6e3d76c8-7494-4738-842f-cf29a3b7f7e1.png)

**Comment by tonghe90** [comment]
@PinTang1999 Thanks for your interest. I would recommend you to train a network with segmentation, offset prediction tasks. Then you can fine-tune this model.
